test-requirements.txt

```diff
@@ -7,3 +7,4 @@ graphviz>=0.8.4
 moto>=1.3.13
 smart-open>=1.8.4
 boto3>=1.10.45
+composeml>=0.1.8
```

Comment:
Will this work with the current release of composeml or does it require the additions from https://github.com/FeatureLabs/compose/pull/121?

Reply:
it does require https://github.com/FeatureLabs/compose/pull/121, but no tests actually depend on it so I've removed the requirement

Reply:
does that mean we should add a test?

Reply:
it gets tested by the notebook but there are no unit tests associated with it.  we could add tests for both pandas and dask approaches passing in compose label times as the cutoff dataframe?

Reply:
i think we should have a unit test